pandas/core/indexes/base.py

```diff
@@ -1366,6 +1367,12 @@ def set_names(self, names, level=None, inplace: bool = False):
                     ( 'cobra', 2018),
                     ( 'cobra', 2019)],
                    names=['species', 'year'])
+        >>> idx.set_names({'kind': 'snake'})
```

Comment:
add a comment on this example (skip a line)

Reply:
I've added 1.2, if this is not ready I will change to 1.3